src/color/creating_reading.js

```diff
@@ -1538,8 +1538,8 @@ function creatingReading(p5, fn){
    * <a href="/reference/p5/colorMode/">colorMode()</a>.
    *
    * @method paletteLerp
-   * @param  {[p5.Color|String|Number|Number[], Number][]} colors_stops color stops to interpolate from
-   * @param  {Number} amt number to use to interpolate relative to color stops
+   * @param {Array.<Array.<(p5.Color|String|Number|Number[]), Number>>} colors_stops color stops to interpolate from
```

Comment:
I added some code earlier in `utils/convert.js` to handle this tuple notation, since `[X, Y]` is the Typescript way to say "array with X in index 0 and Y in index 1" rather than `Array<X, Y>`. I'll just copy it into here and push it since I made the same change elsewhere before :)